backend/command/src/shared/guards/env.guard.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { NotFoundException } from '@nestjs/common';
+import { ENVGuard } from './env.guard';
+
+describe('ENV guard', () => {
+  let guard: ENVGuard;
+  
+  beforeAll(() => {
+    guard = new ENVGuard();
+  });
+  
+  describe('Positive Tests', () => {
+    it('Should pass when NODE_ENV is development', () => {
+      // ARRANGE
+      process.env.NODE_ENV = 'development';
+      // ACT
+      // This does not need an execution context
+      const res = guard.canActivate(null);
+      // ASSERT
+      expect(res).toEqual(true);
```

Comment:
```suggestion
      expect(res).toBe(true);
```
You should use the toBe() function instead of toEqual() when checking boolean values.